templates/post_unsubscribe.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
_might_ be worth having this use `layout.html` for layout (not all pages do this yet, but realistically all of them should

The onbaording survey is a good example of this code, if there's a compelling reason not to do it that way we should talk and see if we can't upgrade the layout template parent-template a little bit.

Reply:
I had taken the initial code from layout.html. I shall check the onboarding survey to see if I have missed something

Reply:
Hmm I should have been more clear. When I say "use layout.html" I don't mean "copy from" I mean "template inherit from".

https://flask.palletsprojects.com/en/stable/patterns/templateinheritance/

Reply:
I'll push what I mean in a PR in just a minute.